tools/vipps-partner-api-postman-collection.json

```diff
@@ -456,7 +317,7 @@
 									"variable": [
 										{
 											"key": "orgno",
-											"value": "451049545",
+											"value": "401884297",
```

Comment:
Did you want to use "{{orgno}}" here, too?

Reply:
Didn't want to bother, since those are error response endpoints, thus "hotfix":P